enterprise-suite/tests/smokecommon

```diff
@@ -16,11 +16,22 @@ error() {
   exit 1
 }
 
+context=minikube
```

Comment:
This is global, so you're repeating this `kubectl config current-context` command for every job.  At best that's inefficient, at worst that's inconsistent results. This should be established once, and kept.